plugins/modules/zos_operator_action_query.py

```diff
@@ -251,7 +292,33 @@ def run_module():
     try:
         new_params = parse_params(module.params)
 
-        cmd_result_a = execute_command("d r,a,s")
+        kwargs = {}
+
+        wait_s = new_params.get("wait_time_s")
+
+        zv = ZOAU_API_VERSION.split(".")
```

Comment:
same as with zos_operator, would prefer using module_utils/zoau_version_checker

Reply:
We will neeed to talk about this... there's a large issue with the zoau_version _checker

Reply:
I'd like to create a new ticket for that change, based on Wednesday's conversation. It works, but I'll make the change so that zoau_version_checker calls the new code, and then this can use zoau_version_checker.

Reply:
#1018

Reply:
Thank you.